crates/rust-lib/src/lib.rs

```diff
@@ -853,6 +878,7 @@ pub trait RustGenerator<'a> {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
This project doesn't use clippy, so mind leaving this out?

Reply:
Yeah I can leave it out